src/systems/userinput/bindings/touchscreen-user.js

```diff
@@ -87,7 +87,7 @@ export const touchscreenUserBindings = addSetsToBindings({
     {
       src: { value: paths.device.touchscreen.isTouchingGrabbable },
       dest: { value: paths.actions.cursor.grab },
-      xform: xforms.copy
+      xform: xforms.rising
```

Comment:
Surprised to see that this was `copy`. I guess it never mattered because we would immediately change action sets?

Reply:
i think typically you'd be tapping and releasing for clicks (so you end up "clicking" a few times) and yes otherwise you're going to change action sets. so it just affected buttons like video chrome and camera